components/monitoring/logging/staging/external-secrets/splunk-log-forwarders-external-secrets.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: external-secrets.io/v1beta1
```

Comment:
Are you sure Kustomize knows how to process this kind of multi-resource file syntax? I haven't seen examples of this elsewhere.

Reply:
@ifireball I tested it with  Kustomize. It worked.

Reply:
ok. not sure where is the resolve thread button...